receiver/k8sobjectsreceiver/README.md

```diff
@@ -63,6 +63,26 @@ with detailed sample configurations in [testdata/config.yaml](./testdata/config.
 
 Follow the below sections to setup various Kubernetes resources required for the deployment.
 
+### Supported Kubernetes objects
+
+The `k8sobjectsreceiver` supports collecting a wide range of standard Kubernetes objects from the API server. Examples of supported objects in the format for `name` parameter:
+
+- `pods`
+- `events`
+- `nodes`
+- `jobs`
+- `cronjobs`
+- `deployments`
+- `services`
+- `configmaps`
+- `secrets`
+- `statefulsets`
+- `replicasets`
+- `daemonsets`
+- `namespaces`
+
+This receiver supports both `pull` and `watch` modes, allowing for flexible and real-time monitoring of these objects. Note that while many standard objects are supported, custom resources defined by CRDs and certain third-party metrics may require additional configuration.
```

Comment:
Are the objects described on the above list the only the receiver supports? Or others can work as well? What's the difference? Can we be more specific to avoid confusion?

Reply:
> Are the objects described on the above list the only the receiver supports? Or others can work as well? What's the difference? Can we be more specific to avoid confusion?

These are the most widely used ones, in the text above it's expressed: `supports collecting a wide range of standard Kubernetes objects from the API server. Examples of supported objects in the format for name parameter`

I guess we can formulate it better to tell the user all standard k8s objects are supported?

Reply:
> CRDs are supported out of the box, not sure what "third-party metrics" are in this context?

You are right, I meant custom resources instead CRDs.

Reply:
You can watch/pull objects described by CRDs, if said CRDs are available in the cluster. That's the only requirement.